tldw_Server_API/app/core/MCP_unified/modules/implementations/quizzes_module.py

```
@@ -435,6 +393,84 @@ def validate_tool_arguments(self, tool_name: str, arguments: dict[str, Any]):
+
+        description = quiz.get("description")
+        if description is not None and (not isinstance(description, str) or len(description) > 2000):
+            raise ValueError("description must be <= 2000 chars")
+
+        workspace_tag = quiz.get("workspace_tag")
+        if workspace_tag is not None and (not isinstance(workspace_tag, str) or len(workspace_tag) > 64):
+            raise ValueError("workspace_tag must be <= 64 chars")
+
+        time_limit_seconds = quiz.get("time_limit_seconds")
+        if time_limit_seconds is not None and (not isinstance(time_limit_seconds, int) or time_limit_seconds < 0):
+            raise ValueError("time_limit_seconds must be a non-negative integer")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)